cache-config/testing/ort-tests/tcdata/todb.go

```diff
@@ -116,7 +105,7 @@ INSERT INTO role (name, description, priv_level) VALUES ('federation','Role for
 `
 	err := execSQL(db, sqlStmt)
 	if err != nil {
-		return fmt.Errorf("exec failed %v", err)
+		return fmt.Errorf("exec failed %w", err)
```

Comment:
Nitpick: you didn't add these, but since you're modifying them, there are a ton of identical errors here. Identical errors are extremely hard to debug, all error strings should be unique. Something like `fmt.Errorf("exec '%s' error: %w", sqlStmt, err)`

Reply:
> all error strings should be unique

That's a very good point that I forget all too often.

Reply:
I reworded it slightly to put the possibly-mult-line query at the end of the error string, though.